7.6/files/config.sh

```diff
@@ -148,6 +148,22 @@ sed -e 's/{{DB_VENDOR}}/'"${DB_VENDOR}"'/' \
     -e 's/{{BIZ_DB_NAME}}/'"${BIZ_DB_NAME}"'/' \
     -i ${BONITA_PATH}/BonitaCommunity-${BONITA_VERSION}-Tomcat-${TOMCAT_VERSION}/setup/database.properties
 
+if [ "$USE_SSL" == "true" ]
+then
+	case "${DB_VENDOR}" in
+		mysql)
+			sed -e "s/mysql\(\.bdm\)\?\.url=.*/&\\&useSSL=true\\&requireSSL=false/" \
```

Comment:
❓ in mysql case, you don't change the `bitronix-resources.properties` as you did for postgres.

Reply:
@laurentleseigneur In mysql case, it was not needed as it is the url from internal.properties which is copied/transform to bitronix-resources.properties for mysql and others; some some reason, the url is not being used for postgresql and this is why extra configuration must be added to bitronix-resources.properties

Reply:
ok, thanks.